ext/snowcrash/src/BlueprintParser.h

```diff
@@ -678,17 +666,14 @@ namespace snowcrash
                 resourceElementSourceMapIt = elementSourceMap->content.elements().collection.begin();
             }
 
-            for (Elements::iterator resourceElementIt = element.content.elements().begin();
-                 resourceElementIt != element.content.elements().end();
-                 ++resourceElementIt) {
-
-                if (resourceElementIt->element == Element::ResourceElement) {
+            for (auto resourceElementIt : element.content.elements()) {
```

Comment:
Not an iterator anymore, consider renaming `resourceElementIt`; for example to `resourceElement`.